compiler/pipes/parse-and-apply-phpdoc.cpp

```diff
@@ -146,6 +146,20 @@ class ParseAndApplyPhpDocForFunction {
 
   void parse_kphp_doc_tag(const php_doc_tag &tag) {
     switch (tag.type) {
+      case php_doc_tag::kphp_color: {
+        const auto raw_colors = split(tag.value, ' ');
```

Comment:
Let's deny multiple colors in one tag. The format of @kphp-color should be:
@kphp-color color [optional comment]
So, if a user wants multiple colors – he writes multiple tags.
It will be much easier to find used colors in a regular code using a simple text search.